mitosheet/mitosheet/public/v1/sheet_functions/misc_functions.py

```diff
@@ -113,8 +115,63 @@ def get_element_type(element):
     return series.apply(get_element_type).astype('str')
 
 
+
+@handle_sheet_function_errors
+@convert_arg_to_series_type(
+    0,
+    'series',
+    'error',
+    'error'
+)
+@convert_arg_to_series_type(
+    1,
+    'bool', 
+    'error', 
+    ('default', True)
+)
+def GETPREVIOUSVALUE(series: pd.Series, condition: pd.Series) -> pd.Series:
```

Comment:
This makes sense for the v1/2 interface. When we move to v3, maybe we should make the following updates: `series: pd.Series` in our new world should probably be `series: pd.DataFrame` since the Excel syntax should ideally look like `A:A` instead of `A0`. 

The question is how would we handle the `condition`. In Excel, it would be something like this: We'll probably want to take whatever we do for COUNTIF and SUMIF

`GETPREVIOUSVALUE(Max_Balances:Max_Balances, "> 0")`